server/portal/apps/workspace/api/views_unit_test.py

```diff
@@ -170,34 +170,6 @@ def test_date_filter(rf, authenticated_user, mock_agave_client):
     assert len(jobs) == 1
 
 
-def test_tray_get_appid_by_spec(authenticated_user, mock_agave_client):
-    compress_01u1 = {
-        'id': 'compress-0.1u1',
-        'name': 'compress',
-        'version': '0.1',
-        'revision': '1'
-    }
-    compress_01u2 = {
-        'id': 'compress-0.1u2',
-        'name': 'compress',
-        'version': '0.1',
-        'revision': '2'
-    }
-    compress_02u1 = {
-        'id': 'compress-0.2u1',
-        'name': 'compress',
-        'version': '0.2',
-        'revision': 1
-    }
-    view = AppsTrayView()
-    mock_agave_client.apps.list.return_value = [compress_01u1, compress_01u2]
-    assert view.getAppIdBySpec(
-        MagicMock(name='compress', version='0.1'), authenticated_user) == 'compress-0.1u2'
-    mock_agave_client.apps.list.return_value = [compress_01u2, compress_02u1]
-    assert view.getAppIdBySpec(
-        MagicMock(name='compress', version='0.1'), authenticated_user) == 'compress-0.2u1'
-
-
 @pytest.mark.skip(reason="Tray endpoint and related tests need to be updated")
 def test_tray_get_app(mocker, client, authenticated_user):
```

Comment:
can this test be dropped?  I see its testing AppsTrayView.getApp that I don't think exists.